app/javascript/app/pages/mitigation/mitigation-effects/summary/summary-styles.scss

```diff
@@ -0,0 +1,67 @@
+@import '~styles/layout.scss';
+
+.columns {
+  @include columns(2);
+
+  align-items: flex-end;
+}
+
+.buttonGroupWrapper {
+  @include column-offset(4);
+
+  width: 90px;
+}
+
+.contentContainer {
+  @include columns(6);
+
+  margin-top: $gutter-padding;
+  position: relative;
+}
+
+.chartContainer {
+  display: flex;
+  justify-content: center;
+
+  :global {
```

Comment:
If we are going to reuse the tooltip styles I think worth it to move to a global scope .scss file avoiding duplicates.